.gitignore

```diff
@@ -46,6 +46,9 @@ MANIFEST
 pip-log.txt
 pip-delete-this-directory.txt
 
+# Geckodriver log
+geckodriver.log
```

Comment:
It would be good to also ignore the `geckodriver.exe` file here since that is largely & will depend on the users system. Also note that `geckodriver.log` was already checked in before you added this & this hasn't removed the file. To get rid of that, you can run `git rm --cached geckodriver.log`

Reply:
Done!